### DISCUSSION TITLE
第2节课作业(16班)

### DISCUSSION THREAD

**Discussion by Yang-Changhui** [InternLM/Tutorial#118] [作业]
第二节课作业请提交在下面评论

**Comment by Eadral** [comment]
https://www.yuque.com/eadral/ruaugi/wm45yncrc0rhwwmf?singleDoc# 《第二节课作业》

**Comment by EnableAsync** [comment]
https://enableasync.github.io/internlm/internlm-02

**Comment by Aitejiu** [comment]
https://www.cnblogs.com/aitejiu/p/17949124

**Comment by future-2002** [comment]
https://blog.csdn.net/Y2483490891/article/details/135424276?spm=1001.2014.3001.5501

**Comment by Aorg** [comment]
https://github.com/Aorg/Assignment/blob/main/lesson2_assignment.md

**Comment by yechenzhi** [comment]
https://docs.qq.com/doc/DZFZMU1VLUmt0emZt

**Comment by Durian-1111** [comment]
**Day02-Homework**
链接：https://xby48gs45u.feishu.cn/docx/Q20idhjpAoLrfdxXf79ctT01n0e?from=from_copylink

**Comment by Star-cre** [comment]
故事截图在2-4：https://zhilengnuan.blog.csdn.net/article/details/135386192?spm=1001.2014.3001.5502
huggingface下载截图在附录，进阶展示图在三和四点的最后：
https://zhilengnuan.blog.csdn.net/article/details/131724182?spm=1001.2014.3001.5502

**Comment by etheral12138** [comment]
https://y1yc0dwhb0o.feishu.cn/docx/AND7dOlztoeebsxQmCicuocCnzg?from=from_copylink

**Comment by morcake** [comment]
https://blog.51cto.com/morcake/9134487

**Comment by andrepan** [comment]
https://www.yuque.com/andrepan/eg7adh/sof250cnu7u355at?singleDoc# 《第二课》

**Comment by zszbyzsz** [comment]
https://juejin.cn/post/7321526403433922595

**Comment by hhhanqian** [comment]
78 【书生·浦语大模型作业（二） - 嗨嗨 | 小红书 - 你的生活指南】 😆 OpT8zUqF3jP9QDk 😆 http://xhslink.com/BDEQXy

**Comment by Dhaizei** [comment]
http://t.csdnimg.cn/XqGaW

**Comment by DavidAlpha007** [comment]
https://github.com/DavidAlpha007/DavidAlpha007/blob/main/Notes%20of%20InternLM

**Comment by 2404589803** [comment]
https://2404589803.github.io/My-datawhale-learning/#/shusheng_puyu_learnning53/ch02

**Comment by springkin** [comment]
https://airy-planarian-ede.notion.site/Demo-8184e282142146a19fa7c868f5cb71b5?pvs=4

**Comment by xuanxiaochenmi** [comment]
https://github.com/xuanxiaochenmi/InternLLM_notes/blob/master/lesson_2/lesson_2.md

**Comment by TianCai19** [comment]
https://codetwice.notion.site/4e53fb982c43419b80413818f6606b6a?pvs=4

**Comment by Bob-Y** [comment]
### 基础作业：

- 使用 InternLM-Chat-7B 模型生成 300 字的小故事（需截图）。
<img width="718" alt="b8b0970688e303b18c3588e982664a6" src="https://github.com/InternLM/tutorial/assets/31715275/b864b9e3-22ae-4aa3-aff2-39652645639c">

<img width="960" alt="7c0c41fe6688f174c0e190e77c89fa4" src="https://github.com/InternLM/tutorial/assets/31715275/d48e1f93-5c9d-4ff8-87b4-051cd9872bac">

- 熟悉 hugging face 下载功能，使用 huggingface_hub python 包，下载 InternLM-20B 的 config.json 文件到本地（需截图下载过程）。
<img width="960" alt="29c1bd3b653e4f3c4a27252a73d98e9" src="https://github.com/InternLM/tutorial/assets/31715275/89216db8-53b3-4bbb-bb41-e853cd3b7009">


### 进阶作业（可选做）

- 完成浦语·灵笔的图文理解及创作部署（需截图）
<img width="960" alt="c91c5541bc38b38b63eb736cf59ee5c" src="https://github.com/InternLM/tutorial/assets/31715275/b76347f2-f772-4d1a-9c92-2f28fae84a2e">

- 完成 Lagent 工具调用 Demo 创作部署（需截图）
<img width="960" alt="33f0e8efc59f53ca973353504c61a63" src="https://github.com/InternLM/tutorial/assets/31715275/e011c8b6-dffd-4320-8f20-b3ca42984c84">
